tests/dtm/test_single_process_restart.py

```diff
@@ -150,24 +181,170 @@ def test_write_during_m0d_restart(self):
+                                  "Expected failures in WRITEs/READs in background during m0d "
+                                  f"restart. Logs collected during m0d restart: {resp[1]}")
+        self.log.info("Step 3: Successfully completed WRITEs/READs in background")
+
+        self.log.info("Step 4: Perform READs-Verify on already written data in Step 1")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=self.iam_user,
+                                                    log_prefix=test_prefix, skipwrite=True,
+                                                    skipcleanup=True, nclients=10, nsamples=30)
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Step 4: Successfully performed READs-Verify on already written data in "
+                      "Step 1")
+
+        self.log.info("ENDED: Verify IOs before and after RC pod m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm0
+    @pytest.mark.tags("TEST-41235")
+    def test_bkt_creation_ios_after_m0d_restart(self):
+        """Verify bucket creation and IOs after m0d restart using pkill."""
+        self.log.info("STARTED: Verify bucket creation and IOs after m0d restart using pkill")
+        test_cfg = DTM_CFG['test_41235']
+
+        self.log.info("Step 1: Create IAM user with name %s", self.s3acc_name)
+        resp = self.rest_obj.create_s3_account(acc_name=self.s3acc_name,
+                                               email_id=self.s3acc_email,
+                                               passwd=S3_CFG["CliConfig"]["s3_account"]["password"])
+        assert_utils.assert_true(resp[0], resp[1])
+        access_key = resp[1]["access_key"]
+        secret_key = resp[1]["secret_key"]
+        self.iam_user = {'s3_acc': {'accesskey': access_key, 'secretkey': secret_key,
+                                    'user_name': self.s3acc_name}}
+        test_prefix = 'test-41235'
+        self.log.info("Step 1: Created IAM user with name %s", self.s3acc_name)
+
+        self.log.info("Step 2: Perform Single m0d Process Restart")
+        self.log.debug("Get IDs of all m0d processes")
+        resp, fids = self.health_obj.hctl_status_get_svc_fids()
+        assert_utils.assert_true(resp, "Failed to get services fids")
+        fids = fids['ioservice']
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            ha_obj=self.ha_obj, pod_prefix=POD_NAME_PREFIX,
+                                            container_prefix=MOTR_CONTAINER_PREFIX,
+                                            process=self.m0d_process, process_ids=fids,
+                                            check_proc_state=True)
+        assert_utils.assert_true(resp, f"Response: {resp} \nhctl status is not as expected")
+        self.log.info("Step 2: m0d restarted and recovered successfully")
+
+        self.log.info("Step 3: Perform WRITEs/READs-Verify/DELETEs with variable sizes objects.")
```

Comment:
will DELETE buckets supported for dtm0, @priyankaborawake can u confirm?

Reply:
Delete buckets will not be supported during m0d restart/recovery.